server/remote/msgraph/event.go

```diff
@@ -49,3 +50,7 @@ func (c *client) TentativelyAcceptEvent(remoteUserID, eventID string) error {
 	}
 	return nil
 }
+
+func (c *client) GetEventsBetweenDates(_ string, start, end time.Time) ([]*remote.Event, error) {
+	return nil, errors.New("not implemented")
```

Comment:
How can we make sure this doesn't get called, both now and in the future?

Reply:
Implementing it is the safest way.